apps/web/components/Address/Address.vue

```diff
@@ -9,16 +9,23 @@
         <AddressDisplay :address="address" />
       </div>
 
-      <SfIconCheckCircle v-if="isDefault" class="flex justify-end text-primary-700 shrink-0 default-address" />
+      <SfIconCheckCircle v-if="isSelected" class="flex justify-end text-primary-700 shrink-0 default-address" />
     </div>
     <div class="actions flex justify-end">
-      <SfButton variant="tertiary" size="sm" class="self-start" @click="$emit('on-edit')">
+      <SfButton variant="tertiary" size="sm" class="self-start" @click="$emit('on-edit')" @click.stop>
```

Comment:
any reason we need ```@click.stop``` ?

Reply:
trying to prevent the @click event from a layer above, to also be triggered by the 'on-edit' and 'on-delete' event. Otherwise when one of these to buttons would be clicked, they would also select another address. So an click on 'edit' would also change the selected address.

Reply:
seems right, haven't thought about this, so i did changed it and will commit soon @csandru-plenty